docs/quickstart-readme.txt

```diff
@@ -0,0 +1,15 @@
+                            DistArray Quickstart
+================================================================================
+This script will setup a new conda environment with DistArray. Depending on the
+prior availability of dependencies and system hardware, this process could take
+anywhere from less than a minute upto a few hours. If installation fails,
+delete the created conda environment [conda env remove -n <env-name>]
+and re-run this script.
+
+Prerequisites for using DistArray quickstart:
+- A working conda installation (Anaconda/Miniconda)
+- OSX only: a working package manager (MacPorts/HomeBrew)
```

Comment:
You should mention that this script *only* works on OS X and Linux (not Windows).